session.py

```diff
@@ -77,11 +77,17 @@ def login(host, email, password):
     arg_parser.add_argument('--user', help='Email address of user making request')
     arg_parser.add_argument('--password', help='The password of the user making the request')
     arg_parser.add_argument('--api-key', help='API key used to authenticate')
+    arg_parser.add_argument('--noverify', action='store_true', help='If passed, the SSL certificate of the host will not be verified')
 
     args = arg_parser.parse_args()
 
     email = args.user
     if email is None:
         email = "dhl-dev@conduce.com"
 
-    print get_session(args.host, email, password)
+    if args.noverify:
+        verify = False
+    else:
+        verify = True
+
+    print get_session(args.host, email, args.password, verify)
```

Comment:
also passed `args.password` here instead of just `password`, as `password` is undefined in this context